src/Microplate/CoordinateSystem1x1.php

```diff
@@ -0,0 +1,19 @@
+<?php declare(strict_types=1);
+
+namespace MLL\Utils\Microplate;
+
+class CoordinateSystem1x1 extends CoordinateSystem
+{
+    /** Duplicates @see CoordinateSystem::positionsCount() for static contexts. */
+    public const POSITIONS_COUNT = 1;
+
+    public function rows(): array
+    {
+        return range('A', 'A');
+    }
+
+    public function columns(): array
+    {
+        return range(1, 1);
```

Comment:
```suggestion
        return ['A'];
    }

    public function columns(): array
    {
        return [1];
```

Dann stellen sich keine Fragen wie sich range verhält, und es ist billiger